service/controller/app/v1/resource/configmap/create.go

```diff
@@ -4,14 +4,18 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/giantswarm/errors/tenant"
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller/context/resourcecanceledcontext"
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 
 	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
 )
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	cr, err := key.ToCustomResource(obj)
```

Comment:
I think when you review your own PR upfront you could recognize that this error here is not handled. Please make sure that such things are covered, as such things occur frequently in your PRs.